manifesto/README.md

```diff
@@ -0,0 +1,8 @@
+# Kyma Elevator Pitch
+
+Kyma is a Cloud Native Application Runtime that gives you the power of Kubernetes but without the hassle of worrying about security, configuration and integration of tools. 
+Kyma does it for you, as it not only combines various components needed for Cloud Native development, but also fills the gaps between them to make them work smoothly together and let you focus on your work. 
```

Comment:
```suggestion
Kyma does it for you, as it not only combines various components needed for cloud-native development but also fills the gaps between these components to make them work smoothly together and let you focus on your work. 
```

Reply:
1. Is there any reason why you capitalize `Cloud-Native`?
2. I guess there should be no comma before `but also` 🤔 See the [reference](https://dictionary.cambridge.org/grammar/british-grammar/not-only-but-also).
3. This sentence is a bit long, that's why I'd change "them" into "these components" to avoid ambiguity.

Reply:
You're totally right! Thanks! 
Ad. 1: the reason was my habit of seeing it mostly in titles :D Thanks!